buildSrc/src/main/kotlin/BuildInfo.kt

```diff
@@ -95,6 +95,11 @@ open class BuildInfo(project: Project) {
     java.lang.Boolean.getBoolean("releaseBuild")
   }
 
+  val hasMuslToolchain: Boolean by lazy {
+    // see "install musl" in .circleci/jobs/BuildNativeJob.pkl
+    File(System.getProperty("user.home"), "staticdeps/bin/musl-gcc").exists()
```

Comment:
Quick suggestion: this is slightly less brittle because this is ultimately what our `native-image` build looks for

```suggestion
    File(System.getProperty("user.home"), "staticdeps/bin/x86_64-linux-musl-gcc").exists()
```